tools/Dockerfile.build.ppc64le

```diff
@@ -0,0 +1,51 @@
+# docker build --build-arg UID=$(id -u) -t nvidia-docker -f Dockerfile.build.ppc64le .
+
+
+FROM ppc64le/golang:1.6.3
+
+ENV NVIDIA_GPGKEY_SUM bd841d59a27a406e513db7d405550894188a4c1cd96bf8aa4f82f1b39e0b5c1c
+ENV NVIDIA_GPGKEY_FPR 889bee522da690103c4b085ed88c3d385c37d3be
+ENV NVIDIA_GDK_SUM 1e32e58f69fe29ee67b845233e7aa9347f37994463252bccbc8bfc8a7104ab5a
+
+RUN curl http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1404/ppc64el/cuda-repo-ubuntu1404_7.5-18_ppc64el.deb -o cudarepo.deb && dpkg -i cudarepo.deb
+
+# There's a bug in binutils that causes a strange compile error, so
+# we need newer binutils and libc
+RUN echo "deb http://httpredir.debian.org/debian stretch main" >> /etc/apt/sources.list
+RUN echo "deb http://httpredir.debian.org/debian stretch-updates main" >> /etc/apt/sources.list
+RUN echo "deb http://security.debian.org stretch/updates main" >> /etc/apt/sources.list
+
+RUN apt-get clean && apt-get update
+RUN apt-get install -y --no-install-recommends --force-yes \
+    # No 6.5 packages for ppc64le
+     cuda-cudart-dev-7-5  cuda-misc-headers-7-5 \
+     && apt-get upgrade -y binutils libc6 libc6-dev libc-bin \
```

Comment:
I would merge the `apt-get upgrade` with the three `echo "deb [...]"` above, and maybe merge them in a single `RUN` statement since all these commands serve a single purpose.